src/locales/ja/battle.json

```
@@ -28,6 +28,7 @@
   "attackMissed": "{{pokemonNameWithAffix}}には　当たらなかった！",
   "attackHitsCount": "{{count}}かい 当たった！",
   "rewardGain": "{{modifierName}}を　手に入れた!",
+  "rewardGainCount": "{{modifierName}}を　{{count}}個　手に入れた!",
```

```suggestion
  "rewardGainCount": "$t(modifierName, {\"count\": {{count}} })を　{{count}}個　手に入れた!",
```

For possible plural handling

DISCLAIMER: Does japanese need this kind of handling for plurals?